libs-scala/test-evidence/tag/src/scala/com/daml/test/evidence/tag/Security.scala

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.test.evidence.tag
+
+import better.files.File
+
+object Security {
+
+  /** Security-relevant information on a test-suite level. */
+  trait SecurityTestSuite {
+
+    /** The layer that the security test suite tests, such as on the network/API level or the ledger model. */
+    def securityTestLayer: SecurityTestLayer
```

Comment:
FYI: This field may need to be revised. Usually a suite mixes test cases from different layers. So it gets hard to choose a single layer that describes all tests.

We should have a wider discussion about the data model encoded here at some point. So that we can commit to the best possible model before annotating tons of tests.

Reply:
@skisel-da please incorporate this observation in form of a todo and follow up with a discussion after this PR is merged